scopes/defender/linter/lint.cmd.ts

```diff
@@ -117,17 +141,39 @@ function toJsonLintResults(results: EnvsExecutionResult<LintResults>): JsonLintD
   let totalFixableErrorCount = 0;
   let totalFixableWarningCount = 0;
   let totalWarningCount = 0;
+  let totalComponentsWithErrorCount = 0;
+  let totalComponentsWithFatalErrorCount = 0;
+  let totalComponentsWithFixableErrorCount = 0;
+  let totalComponentsWithFixableWarningCount = 0;
+  let totalComponentsWithWarningCount = 0;
+
   const newResults = results.results.map((res) => {
     const resultsWithoutComponent = res.data?.results.map((result) => {
       return Object.assign({}, { componentId: result.component.id }, omit(result, ['component']));
     });
 
     if (res.data) {
-      totalErrorCount += res.data.totalErrorCount ?? 0;
-      totalFatalErrorCount += res.data.totalFatalErrorCount ?? 0;
-      totalFixableErrorCount += res.data.totalFixableErrorCount ?? 0;
-      totalFixableWarningCount += res.data.totalFixableWarningCount ?? 0;
-      totalWarningCount += res.data.totalWarningCount ?? 0;
+      if (res.data.totalErrorCount){
+        totalErrorCount += res.data.totalErrorCount ?? 0;
+        totalComponentsWithErrorCount += res.data.totalComponentsWithErrorCount ?? 0;
```

Comment:
shouldn't this be `+=1`?
```suggestion
        totalComponentsWithErrorCount += 1;
```

Reply:
no, at this point is in a format of total components for each env.
and we are now summing it into the full total.